src/app/components/ArchiveComponent.tsx

```diff
@@ -3,8 +3,10 @@ import { useDropzone } from 'react-dropzone';
 import JSZip from 'jszip';
 import VirtualInstallation from './VirtualInstallation';
 import Encoding from 'encoding-japanese';
-import { Archive } from 'libarchive.js/main.js';
-import { CompressedFile } from 'libarchive.js/src/compressed-file';
+import { Archive } from 'libarchive.js';
+import type { CompressedFile } from 'libarchive.js/src/compressed-file';
```

Comment:
Happy Halloween!🎃

```suggestion
import type { CompressedFile } from 'libarchive.js/dist/build/compiled/compressed-file';
```
Since skiplibcheck is true, you can skip type checking by importing .d.ts instead of .ts.

Reply:
Happy Halloween!! 🎃🦇🕸
And thanks!

Reply:
So I think the patch for libarchive.js can be removed.

Reply:
Fixed.